tests/lib/eslint.js

```diff
@@ -1837,74 +1837,213 @@ describe("eslint", function() {
-                "alert('test');" // here
-            ].join("\n");
-            var config = {
-                rules: {
-                    "no-alert": 1,
-                    "no-console": 1
-                }
-            };
+            it("should report a violation", function() {
+                var code = [
+                    "alert('test'); // eslint-disable-line no-alert",
+                    "console.log('test'); // eslint-disable-line no-console",
+                    "alert('test');" // here
+                ].join("\n");
+                var config = {
+                    rules: {
+                        "no-alert": 1,
+                        "no-console": 1
+                    }
+                };
 
-            var messages = eslint.verify(code, config, filename);
-            assert.equal(messages.length, 1);
+                var messages = eslint.verify(code, config, filename);
+                assert.equal(messages.length, 1);
 
-            assert.equal(messages[0].ruleId, "no-alert");
-        });
+                assert.equal(messages[0].ruleId, "no-alert");
+            });
 
-        it("should report a violation", function() {
-            var code = [
-                "alert('test'); // eslint-disable-line no-alert",
-                "alert('test'); /*eslint-disable-line no-alert*/" // here
-            ].join("\n");
-            var config = {
-                rules: {
-                    "no-alert": 1
-                }
-            };
+            it("should report a violation", function() {
+                var code = [
+                    "alert('test'); // eslint-disable-line no-alert",
+                    "alert('test'); /*eslint-disable-line no-alert*/" // here
+                ].join("\n");
+                var config = {
+                    rules: {
+                        "no-alert": 1
+                    }
+                };
 
-            var messages = eslint.verify(code, config, filename);
-            assert.equal(messages.length, 1);
+                var messages = eslint.verify(code, config, filename);
+                assert.equal(messages.length, 1);
 
-            assert.equal(messages[0].ruleId, "no-alert");
+                assert.equal(messages[0].ruleId, "no-alert");
+            });
+
+            it("should not report a violation", function() {
+                var code = [
+                    "alert('test'); // eslint-disable-line no-alert",
+                    "console('test'); // eslint-disable-line no-console"
+                ].join("\n");
+                var config = {
+                    rules: {
+                        "no-alert": 1,
+                        "no-console": 1
+                    }
+                };
+
+                var messages = eslint.verify(code, config, filename);
+                assert.equal(messages.length, 0);
+            });
+
+            it("should not report a violation", function() {
```

Comment:
Didn't want to mess with the previously existing tests too much, but wanted to ensure we have a test for multi rule `eslint-disable-line` and for when no rule is specified at all, since neither were covered by tests previously.